recipes/llm-voice-assistant/ios/README.md

```diff
@@ -27,11 +27,7 @@ Copy your model file to the device.
 
 ## Usage
 
-1. Install the dependencies using `CocoaPods`:
-
-```console
-pod install
-```
+1. Install the dependencies using `Swift Package Manager`:
 
 2. Open the `LLMVoiceAssistantDemo.xcworkspace` in XCode
```

Comment:
Since SPM package install is automatic and there is no longer an xcworkspace file, I think we need to change this to `1. Open the LLMVoiceAssistantDemo.xcodeproj in XCode`